src/tools/debugger/session/Breakpoints.java

```diff
@@ -74,6 +74,7 @@ public synchronized void addOrUpdate(final LineBreakpoint bId) {
       WebDebugger.log("LineBreakpoint: " + bId);
       bp = Breakpoint.newBuilder(bId.getURI()).
           lineIs(bId.getLine()).
+          steppingLocation(SteppingLocation.BEFORE_STATEMENT).
```

Comment:
This one: https://github.com/smarr/truffle/commit/a4b4d2b942578d9b8b2bd9b19fb9e5836b77703f#diff-6be44cfcbe543486a933eb106a501f8aR775